### PR TITLE
New test cases added

### PR DESCRIPTION
**Pull request by ChamodDamitha**

### CODE REVIEW

File: component/src/test/java/org/wso2/extension/siddhi/execution/approximate/count/CountTestCase.java [hunk 1]
```diff
@@ -292,6 +291,175 @@ public void testApproximateCount_9() throws InterruptedException {
...
+                eventArrived = true;
+            }
+        });
+
+        InputHandler inputHandler = siddhiAppRuntime.getInputHandler("inputStream");
+        siddhiAppRuntime.start();
+
+        for (int j = 0; j < totalEventsSent; j++) {
+            inputHandler.send(new Object[]{j % noOfUniqueEvents});
+            Thread.sleep(1);
```
Comment (NisalaNiroshana) [inline component/src/test/java/org/wso2/extension/siddhi/execution/approximate/count/CountTestCase.java]: Is it really making any difference here? You have put it 1 millisecond here. Does it make any effect on the test case? I think we can remove this. WDYT ? Please refer the other places also

File: component/src/test/java/org/wso2/extension/siddhi/execution/approximate/count/CountTestCase.java [hunk 2]
```diff
@@ -292,6 +291,175 @@ public void testApproximateCount_9() throws InterruptedException {
...
+                    }
+                }
+                eventArrived = true;
+            }
+        });
+
+        InputHandler inputHandler = siddhiAppRuntime.getInputHandler("inputStream");
+        siddhiAppRuntime.start();
+
+        for (int j = 0; j < totalEventsSent; j++) {
```
Comment (NisalaNiroshana) [inline component/src/test/java/org/wso2/extension/siddhi/execution/approximate/count/CountTestCase.java]: Shall we use a meaningful variable name without using 'j' ?  Please look other places which use 'j' as a variable

File: component/src/test/java/org/wso2/extension/siddhi/execution/approximate/count/CountTestCase.java (hunk 1 above)
Comment (ChamodDamitha) [reply component/src/test/java/org/wso2/extension/siddhi/execution/approximate/count/CountTestCase.java]: No effect. I will remove it

File: component/src/test/java/org/wso2/extension/siddhi/execution/approximate/count/CountTestCase.java (hunk 2 above)
Comment (ChamodDamitha) [reply component/src/test/java/org/wso2/extension/siddhi/execution/approximate/count/CountTestCase.java]: I will change it

**Review by dnwick — APPROVED** [review]

**Pull request merged**